composer.json

```diff
@@ -18,42 +18,87 @@
 		}
 	},
 	"require": {
-		"php": ">=7.2",
+		"php": ">=7.2.5",
+		"ext-ctype": "*",
 		"ext-dom": "*",
+		"ext-iconv": "*",
 		"ext-json": "*",
 		"ext-libxml": "*",
 		"ext-mbstring": "*",
 		"ext-pdo": "*",
 		"ext-sqlite3": "*",
 		"ext-zip": "*",
+		"composer/package-versions-deprecated": "^1.11",
+		"doctrine/annotations": "^1.10",
+		"doctrine/doctrine-bundle": "^2.1",
 		"guzzlehttp/guzzle": "^6.0",
 		"monolog/monolog": "^2.0",
-		"symfony/process": "^4.3",
-		"wikimedia/html-formatter": "^1.0"
+		"symfony/console": "5.1.*",
+		"symfony/dotenv": "5.1.*",
+		"symfony/flex": "^1.3.1",
+		"symfony/framework-bundle": "5.1.*",
+		"symfony/process": "5.1.*",
+		"symfony/yaml": "5.1.*",
+		"twig/extra-bundle": "^2.12 || ^3.0",
+		"twig/twig": "^2.12 || ^3.0",
+		"wikimedia/html-formatter": "^1.0",
+		"wikimedia/toolforge-bundle": "^1.0"
```

Comment:
wikimedia/toolforge-bundle 1.0.0 requires ext-intl

You need to add 
```
libicu-dev \
...
docker-php-ext-install intl \
```
to the Dockerfile to satisfy the requirements (I think)